jsonrpc/helper.go

```diff
@@ -35,9 +35,6 @@ func GetNumericBlockNumber(number BlockNumber, store latestHeaderGetter) (uint64
 	case EarliestBlockNumber:
 		return 0, nil
 
-	case PendingBlockNumber:
-		return 0, ErrPendingBlockNumber
```

Comment:
Seems like `ErrPendingBlockNumber` can be deleted, it isn't used anymore?

Reply:
https://github.com/0xPolygon/polygon-edge/pull/974/commits/8a49a83807191f89fffce9adac5ea44c636a3449